### PR TITLE
Add Tkinter Workshop

### PR DESCRIPTION
**Pull request by bezlin6mechminerz**
@MatthewStanciu I have reduced the details of the explanation.

### CODE REVIEW

**Comment by MatthewStanciu** [comment]
One small change: can you rename the folder from `Tkinter Workshop` to `tkinter_workshop`? Just to follow the format of the other workshop folders

File: workshops/Tkinter Workshop/README.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+window.config(bg="black")
+tk.Label(window, font=("Helvetica", 60, "bold"),
+ text="QUOTES HUB", bg="black", fg="white").pack()
+tk.Button(window,command=lambda:clicked(random.choice(quotelist)),text="Open",bg="aqua",fg="black",width=20).place(x=315,y=300)
+window.mainloop()
+
+```
+
+![run it](https://cloud-5m2nwfs8r.vercel.app/ezgif.com-video-to-gif-4.gif)
+
```
Comment (MatthewStanciu) [inline workshops/Tkinter Workshop/README.md:201]: Here, can you include a screenshot or gif of you running the program and the final demo apepars?

**Comment by MatthewStanciu** [comment]
Ping me on Slack (@matthew) once you’ve made these changes and I’ll re-review. I think it’s getting really close to being ready to merge!

File: workshops/Tkinter Workshop/README.md (hunk 1 above)
Comment (bezlin6mechminerz) [reply workshops/Tkinter Workshop/README.md:201]: ok

**Comment by MatthewStanciu** [comment]
Hey, Bezlin. After reviewing this workshop again, I don't think I can accept it. It's difficult to pinpoint exactly which rules the workshop isn't following—mostly I think this workshop just doesn't make a compelling project that most Hack Clubs would be able to use to run a great club meeting.

I recognize that you've put a lot of effort into getting this workshop in a good spot, from spending lots of time writing it and making changes, to getting people to agree to test it in a meeting with you. I don't want your effort to go unnoticed, so if you fill out the form at https://hack.af/bountypayment, I can send you $50 USD.

Happy to talk through more details if you want. Just DM me. Thanks so much for putting in the effort to submit this workshop 🙌

**Comment by bezlin6mechminerz** [comment]
Thank you! But I hope you would have said that before long wait. But no problem for me it at least helped me to learn some grammar.

**Comment by bezlin6mechminerz** [comment]
Do I have any chance of other workshops?

**Pull request closed without merging**